src/akeneo/README.md

```diff
@@ -28,6 +28,8 @@ git commit -m "Initial commit"
 - A valid subscription with Akeneo to gain access to the enterprise edition codebase.
 - A dedicated user in the Akeneo Portal for the purposes of installing the enterprise edition with composer.
   https://help.akeneo.com/portal/articles/get-akeneo-pim-enterprise-archive.html
+- A public and private SSH keypair generated specifically for the dedicated user in the Akeneo Portal, with the public
+  key associated with the dedicated user.
```

Comment:
```suggestion
A public and private SSH keypair generated, with the public key associated with the dedicated user in the Akeneo Portal.
```
Previously it reads to me as if `specifically` means it can't be used for other purposes as well

Reply:
You can associate multiple ssh keys in the portal, so I don't see the harm in having a key just for pulling down code

Reply:
the opposite I mean, if another repository needs an ssh key (e.g. just one github repository's deploy key)

Reply:
Ah I see, we could:
```suggestion
- A public and private SSH keypair generated for the purposes of installing the codebase, with the public
  key associated with the dedicated user in the Akeneo Portal.
```

Reply:
ok